src/helpers.js

```diff
@@ -1,3 +1,3 @@
-import { createBrowserHistory } from 'history';
+import { createBrowserHistory } from 'history'
 
-export const history = createBrowserHistory();
```

Comment:
Why do you removed here semicolons? Because as I see, other exports and imports has semicolons.